template/deployments/helm/charts/templates/deployment.yaml

```diff
@@ -31,8 +31,8 @@ spec:
         - name: {{ .Chart.Name }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
-          image: "{{ .Values.imageKey.repository }}"
-          imagePullPolicy: {{ .Values.imageKey.pullPolicy }}
+          image: "{{ .Values.image.repository }}"
```

Comment:
this needs to be updated to include the tag

`image: {{ .Values.image.repository }}:{{ .Values.image.tag }}`

Reply:
changing this would make the IMAGENAME field no longer include the tag, so we'll need to add that to the release notes as a potentially breaking change

Reply:
why would that be a breaking change?

Reply:
Why are we removing the imagePullPolicy?

Reply:
@sabbour ive checked to make sure the pullpolicy isn't impacted now